admin-ui/app/template/inner-layout/inner-header/InnerHeaderComponent.tsx

```diff
@@ -50,8 +50,6 @@ const InnerHeaderComponent = ({
     fetchData();
```

Comment:
what is fetch data , you have to use fix define fetch strategy for all the fetch call, and it must be asynch